packages/dashboard/src/components/assetModelSelection/assetModelSelection.tsx

```diff
@@ -11,8 +20,33 @@ type AssetModelSelectionOptions = {
 export const AssetModelSelection = ({ client }: AssetModelSelectionOptions) => {
   const { assetModelId, assetIds, hasModelBasedQuery } = useModelBasedQuery();
   const selectedAssetId = assetIds?.at(0);
+  const assetModelSelectionControlId = 'asset-model-select-dropdown';
+
+  const divider = {
+    borderLeft: `solid ${spaceScaledXxxs} ${colorBorderDividerDefault}`,
+    height: spaceStaticXxl,
+    margin: `0 ${spaceScaledS}`,
+  };
+
+  const Divider = () => <div style={divider} />;
 
   if (!hasModelBasedQuery || !assetModelId) return null;
 
-  return <AssetModelSelect assetModelId={assetModelId} selectedAssetId={selectedAssetId} client={client} />;
+  return (
+    <div className='asset-model-selection'>
+      <h4 style={{ color: colorTextBodyDefault }}>
+        <label htmlFor={assetModelSelectionControlId}>Assets</label>
+      </h4>
+      <Divider />
+      <div style={{ width: '300px' }}>
```

Comment:
is it always the case that this should only be 300px wide? Is there any other way we can make this more responsive?